packages/react-scripts/package.json

```diff
@@ -56,8 +56,14 @@
     "eslint-webpack-plugin": "^2.1.0",
     "file-loader": "6.1.1",
     "fs-extra": "^9.0.1",
+    "fsevents": "^2.1.3",
```

Comment:
Can't find where we are using it? Do we really need it?

Reply:
I'm not sure why npm put that in there; I'll remove it and see if tests pass

```suggestion
```

Reply:
yep that was a good call @\nurlansu; the test passes